tests/console/check_system_info.pm

```diff
@@ -35,6 +39,25 @@ sub run {
         assert_script_run("grep -w $milestone_version /etc/issue");
     }
     assert_script_run('cat /etc/os-release');
+
+    my $myaddons = "";
+    $myaddons    = get_var('SCC_ADDONS');
+    $myaddons    = $myaddons . ",base,serverapp" if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
+    $myaddons    = $myaddons . ",base,desktop,we,python2" if (is_sle('15+') && check_var('SLE_PRODUCT', 'sled'));
+    $myaddons    = $myaddons . ",base,serverapp,desktop,dev,lgm,python2,wsm" if (is_sle('<15', get_var('HDDVERSION')) && is_sle('12+', get_var('HDDVERSION')) && is_sle('15+', get_var('UPGRADE_TARGET_VERSION'))); 
+    
+    # After upgrade, system don't include ltss extension
+    $myaddons         =~ s/ltss,?//g;
+    my @addons        = grep { defined $_ && $_ } split(/,/, $myaddons);
+    my @unique_addons = uniq @addons;
+    foreach my $addon (@unique_addons) {
+        my $name = get_addon_fullname($addon);
+        record_info("$addon module fullname: ", $name);
+        $name = "sle-product-we"      if ($name =~ /we/);
+        $name = "SLE-Module-DevTools" if ($name =~ /development/);
```

Comment:
Yutao,

Why you only set nick name for we and dev?

Reply:
As zypper lr full name is different with SUSEConnect command output. Our get_addon_fullname function use SUSEconnect command output. As this check zypper lr output information, it need change name follow this information.

Reply:
Great, can you please add some comments, then we can get more clear info